modules/cert-manager-certificate-ingress.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/c
+
+:_mod-docs-content-type: PROCEDURE
+[id="cert-manager-certificate-ingress_{context}"]
+= Creating certificates for Ingress Controller
+
+
+.Prerequisites
+
+* You have access to the cluster with `cluster-admin` privileges.
+* You have installed {cert-manager-operator}.
+
+.Procedure
+
+. Create an issuer. For more information, see "Configuring an issuer" in "Additional Resources" section.
+
+. Create a certificate:
+
+.. Create a YAML file, for example, `certificate.yaml`, that defines the `Certificate` object:
++
+.Example `certificate.yaml` file
++
+[source, yaml]
+----
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: <tls_cert> #<1>
+  namespace: openshift-ingress
+spec:
+  isCA: false
+  commonName: "<common_name>" #<2>
+  secretName: <tls_cert> #<3>
+  dnsNames:
+  - "*.new-ingress.<sub.domai.ns-domain-x.y.z.com>" #<4>
+  issuerRef:
+    name: <issuer_name> #<5>
+    kind: Issuer
+----
+<1> Provide a name for the certificate.
+<2> Specify the common name (CN).
+<3> Specify the name of the secret to create that contains the certificate.
+<4> Specify the DNS name of the ingress.
+<5> Specify the name of the issuer.
+
+.. Create the `Certificate` object by running the following command:
++
+[source, terminal]
+----
+$ oc create -f certificate.yaml
+----
+
+. Replace the default ingress certificate. For more information, see "Replacing the default ingress certificate" section in "Additional resources".
++
+once certificate is ready, application workloads on your cluster can start using the generated certificate secret.
```

Comment:
I am hoping this is a typo, as same sentence is repeated after `Verification` step.

Reply:
yes, that's a typo. Fixed.